recipes/sqlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version="3.11.1" %}
+{% set version_split=version.split(".") %}
+{% set major=version_split[0] %}
+{% set minor=version_split[1]|int * 100 %}
+{% set bugfix=version_split[2]|int * 100 %}
+{% set version_coded=(major ~ (minor|string)[:3] ~ (bugfix|string)[:3]) %}
+
+package:
+  name: sqlite
+  version: {{ version }}
+
+source:
+  fn: sqlite-autoconf-{{ version_coded }}.tar.gz
+  url: https://www.sqlite.org/2016/sqlite-autoconf-{{ version_coded }}.tar.gz
+  sha1: c4b4dcd735a4daf5a2e2bb90f374484c8d4dad29
+
+build:
+  number: 0
+  features:
+    - vc9    # [win and py27]
+    - vc10   # [win and py34]
+    - vc14   # [win and py35]
+
+requirements:
+  build:
+    # python is for lining up vc feature (runtime)
+    - python      # [win]
+
+test:
+  requires:
+    - python  2.7*     # [win and py27]
+    - python  3.4*     # [win and py34]
+    - python  3.5*     # [win and py35]
+  commands:
+    - sqlite3 --version
+
+about:
+  home: http://www.sqlite.org/
+  license: Public-Domain (http://www.sqlite.org/copyright.html)
+  #license_family: Public-Domain
```

Comment:
Sure. We have no complaints with this metadata, but we don't enforce it either.

As long as it isn't left commented that is fine. Though this is not a blocker for merging.